### PR TITLE
Small refactor from last PR

### PR DESCRIPTION
**Pull request by JackyChiu**
- Renamed `leaveHandler` -> `quitHandler` for clarity
- Added blocks to switch statement
Why?
Blocks will keep variables created in scope. (Not currently need but for the best)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -80,31 +81,36 @@ function messageHandler(message) {
...
+        }
 
-        default:
+        default: {
              socket.emit('chat', {
                 name: username,
                 date: new Date(),
                 message: message
             })
+            break
```
Comment (vikrambombhi) [inline app.js]: break not needed here

**Review by vikrambombhi — CHANGES_REQUESTED** [review]

File: app.js (hunk 1 above)
Comment (JackyChiu) [reply app.js]: Good catch 👍

**Review by vikrambombhi — APPROVED** [review]

**Comment by vikrambombhi** [comment]
All good

File: app.js [hunk 2]
```diff
@@ -80,31 +81,35 @@ function messageHandler(message) {
...
+        case ':gr': {
             getAndPrintRooms()
                 .then(() => {
                     Input.setPrompt()
                 })
             break
+        }
 
-        case ':help':
+        case ':help': {
```
Comment (BenEmdon) [inline app.js:94]: switch scope? Sweeet 👏

Comment (JackyChiu) [reply app.js:94]: Yep 💯

**Pull request merged**